src/MuTalk-Tests/RandomOperatorMutantSelectionStrategyTest.class.st

```diff
@@ -0,0 +1,6 @@
+Class {
+	#name : 'RandomOperatorMutantSelectionStrategyTest',
+	#superclass : 'RandomMutantSelectionStrategyTest',
+	#category : 'MuTalk-Tests',
+	#package : 'MuTalk-Tests'
+}
```

Comment:
This class is empty, is that right?

Reply:
Yes, because I didn't really figure out any test that could be meaningful to add. But it still runs the same tests as the superclass.